pkgs/servers/home-assistant/default.nix

```diff
@@ -411,12 +411,6 @@ python.pkgs.buildPythonApplication rec {
 
   # leave this in, so users don't have to constantly update their downstream patch handling
   patches = [
-    (fetchpatch {
```

Comment:
Remove the input.

Reply:
Pre-commit hooks when. Statix could do that job just as well. 🙄